@vates/nbd-client/.USAGE.md

```diff
@@ -0,0 +1,19 @@
+### `new NdbClient({address, exportname, secure = true, port  = 10809})`
+
+create a new nbd  client
+
+
+```js
+
+  import NbdClient from '@vates/nbd-client'
+  const client = new NbdClient({
+    address: 'MY_NBD_HOST',
+    exportname: 'MY_SECRET_EXPORT',
+    secure: true,
+  })
+
+  await client.connect()
+  const block = await client.readBlock(blockIndex, BlockSize)
+  await client.disconnect()
```

Comment:
If connect/disconnect are necessary, why not use a factory function which returns a disposable?

Reply:
Why do you think about this?

Reply:
I am still thinking about it. Nbd will be an optionnal feature in backup, it's not that bad if it fails for a backup, we'll fall back to the export method
On the other hand  it will be mandatory for backup resuming. 
What do you think of making another that will expose the disposable for convenience, with the default export still the "raw" client ?

Reply:
I don't understand, I don't see any case where we want to leave the client connected if something else failed :thinking: 

We could indeed add a dedicated method for this but I'm not fond of having multiple ways to do the same things without a good reason :slightly_smiling_face: